src/main/java/org/cascadebot/cascadebot/commands/music/EqualizerCommand.java

```diff
@@ -0,0 +1,214 @@
+package org.cascadebot.cascadebot.commands.music;
+
+import com.sedmelluq.discord.lavaplayer.filter.equalizer.Equalizer;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.User;
+import org.apache.commons.lang3.StringUtils;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.ISubCommand;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.music.CascadeLavalinkPlayer;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.buttons.Button;
+import org.cascadebot.cascadebot.utils.buttons.ButtonGroup;
+
+import java.awt.Color;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class EqualizerCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (!CascadeBot.INS.getMusicHandler().isLavalinkEnabled()) {
+            context.getTypedMessaging().replyDanger(context.i18n("commands.equalizer.not_lavalink"));
+            return;
+        }
+        CascadeLavalinkPlayer player = (CascadeLavalinkPlayer) context.getMusicPlayer();
```

Comment:
Could you potentially do a check to make sure music player is an instance of the cascade one? Just to sanity check